AppCenter/AppCenter/Internals/Model/MSAbstractLog.m

```diff
@@ -61,7 +62,7 @@ - (BOOL)isEqual:(id)object {
     return NO;
   }
   MSAbstractLog *log = (MSAbstractLog *)object;
-  return ((!self.type && !log.type) || [self.type isEqualToString:log.type]) &&
+  return ((!self.tag && !log.tag) || [self.tag isEqual:log.tag]) && ((!self.type && !log.type) || [self.type isEqualToString:log.type]) &&
```

Comment:
I saw Android also updated equals/hashCode methods with tag but I wonder if this is the expected behavior for comparing logs. If logs are same but have different tags, do we really consider the logs as different?

Reply:
I would think yes, but I'd be open to hearing why that shouldn't be the case.  Reasoning?